packages/kotti-ui/source/kotti-filter/utils.ts

```diff
@@ -57,48 +57,54 @@ export const getFilterInitialState = (
 }
 
 export const getOperationOptions = (
-	filterType: Kotti.Filter.FilterType,
+	column: Kotti.Filter.Column,
 ): Kotti.FieldSingleSelect.Props['options'] => {
 	const translations = useTranslationNamespace('KtFilter')
-	switch (filterType) {
+	switch (column.type) {
 		case Kotti.Filter.FilterType.BOOLEAN:
-			return Object.values(Kotti.Filter.Operation.Boolean).map((operation) => ({
-				label: translations.value.boolean[operation],
-				value: operation,
-			}))
+			return (column.operations as Kotti.Filter.Operation.Boolean[]).map(
```

Comment:
if you go with the generics, you may not need those type casts

Reply:
this code was already refactored, there's no type casting anymore